en/mkdocs.yml

```diff
@@ -722,7 +722,24 @@ nav:
             - API Manager Analytics:
                 - Overview of API Analytics: observe/api-manager-analytics/overview-of-api-analytics.md
                 - Publishing Events to Analytics Cloud: observe/api-manager-analytics/publishing-events-to-analytics-cloud.md
-                - Viewing Statistics in Cloud: observe/api-manager-analytics/viewing-stats-in-cloud.md
+                - Configuring APIM Analytics:
```

Comment:
What sort of content are we expecting under Publishing events to Analytics cloud

Reply:
We had some content added for this for Alpha - https://apim.docs.wso2.com/en/4.0.0-dev/observe/api-manager-analytics/publishing-events-to-analytics-cloud/

Is this not relevant anymore?

Reply:
The content u have pointed will go under Configure APIM Analytics -> Configuring synapse gateway. So either we can rename 'Configuring APIM analytics' to 'Publishing Events to Analytics cloud' or simple drop 'Publishing Events to Analytics cloud' topic

Reply:
Fixed.